ta/ta.mk

```diff
@@ -5,7 +5,15 @@ sm := $(ta-target)
 sm-$(sm) := y
 
 # Setup compiler for this sub module
-CROSS_COMPILE_$(sm)	?= $(CROSS_COMPILE)
+ifeq ($(sm),ta_arm32)
```

Comment:
The change in `core/core.mk` above would also better be in `core/arch/arm/arm.mk` I suppose. I will update the PR.